### DISCUSSION TITLE
Terraria on Raspberry pi

### DISCUSSION THREAD

**Discussion by JOLIGEOLIFANJTES** [Pryaxis/TShock#2956] [General]
I am trying to run a Server on my raspberry pi 4 (4GB ram). but i can not get the newer versions to work. i found one old one that works but it would be nice to have a newer one working, this is the old one: wget https://github.com/Pryaxis/TShock/releases/download/v4.3.26/tshock_4.3.26.zip.

What i would like to know what new version should be working on raspberry pi?

**Comment by sgkoishi** [comment]
You need a 64-bit OS installed, and run the `TShock.Installer`.
`mono` is no longer needed.

**Reply by JOLIGEOLIFANJTES** [reply]
What way do you need to run Tschock.installer because when i try to run it, it asks me what application to run it with.

Thanks already!!!

**Reply by sgkoishi** [reply]
It is executable itself. Open your terminal, go to where it is and type in `./TShock.Installer`.
